scripts/acnv_cromwell_tests/case_input_wgs.tsv

```diff
@@ -1,2 +1,2 @@
-T1	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-t1-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-t1-chr20-downsampled.deduplicated.bam.bai	N1	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n1-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n1-chr20-downsampled.deduplicated.bam.bai
-T2	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-t2-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-t2-chr20-downsampled.deduplicated.bam.bai	N2	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n2-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n2-chr20-downsampled.deduplicated.bam.bai
+T1	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143-t1-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143-t1-chr20-downsampled.deduplicated.bam.bai	N1	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n1-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n1-chr20-downsampled.deduplicated.bam.bai
+T2	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143-t2-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143-t2-chr20-downsampled.deduplicated.bam.bai	N2	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n2-chr20-downsampled.deduplicated.bam	/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/HCC1143_BL-n2-chr20-downsampled.deduplicated.bam.bai
```

Comment:
Oops, just checked their headers - they are from the HCC1143 sample, not HCC1143BL. I have no idea why I renamed them...